specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/examples/PotentialOutages_Get.json

```diff
@@ -0,0 +1,53 @@
+{
+  "parameters": {
+    "potentialOutageName": "default",
+    "api-version": "2021-01-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "10000064644594",
```

Comment:
ID should be the standard ARM resource ID. `type` and `name` are also required properties.

https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#get-resource

Reply:
renamed the field to address the ambiguity here